docs/index.md

```diff
@@ -1,22 +1,30 @@
 # MCUboot
 
+MCUboot is a secure bootloader for 32-bit MCUs.
+
 ## Overview
 
-MCUboot is a secure bootloader for 32-bit MCUs.   The goal of MCUboot is to
-define a common infrastructure for the bootloader, system flash layout on
-microcontroller systems, and to provide a secure bootloader that enables
-easy software upgrade.
+The goal of MCUboot is to define a common infrastructure for the bootloader,
```

Comment:
Do we still want to mention 32-bits? We are definitely not going to be supporting 8 or 16 bit MCUs.

Reply:
removed the mention

Reply:
I guess I didn't word that very clearly. I meant to say that we should probably continue to mention 32-bits, as we definitely don't support smaller microcontrollers, and we haven't really tried porting it to 64 bits.

Reply:
Ok, I guess I jumped the gun here. Change reverted.